iamidentityv1/iam_identity_v1.go

```diff
@@ -2260,13 +2269,13 @@ func (iamIdentity *IamIdentityV1) CreateReportWithContext(ctx context.Context, c
 	}
 
 	pathParamsMap := map[string]string{
-		"account_id": *createReportOptions.AccountID,
+		"account-id": *createReportOptions.AccountID,
 	}
 
 	builder := core.NewRequestBuilder(core.POST)
 	builder = builder.WithContext(ctx)
 	builder.EnableGzipCompression = iamIdentity.GetEnableGzipCompression()
-	_, err = builder.ResolveRequestURL(iamIdentity.Service.Options.URL, `/v1/activity/accounts/{account_id}/report`, pathParamsMap)
+	_, err = builder.ResolveRequestURL(iamIdentity.Service.Options.URL, `/v1/activity/accounts/{account-id}/report`, pathParamsMap)
```

Comment:
looks like this path param was changed from using snake-case (which is correct) to using kebab case (which is incorrect).  Why the change?   Did you run the openapi-validator on the updated API definition???

Reply:
oh yes, I did generate using the latest API definition, was wondering which could be correct. I will change it back to snake_case.

Reply:
@padamstx updated :)